regexec.c

```diff
@@ -4412,7 +4412,7 @@ S_dump_exec_pos(pTHX_ const char *locinput,
 
         const STRLEN tlen=len0+len1+len2;
         Perl_re_printf( aTHX_
-                    "%4" IVdf " <%.*s%.*s%s%.*s>%*s|%4u| ",
+                    "%4" IVdf " <%.*s%.*s%s%.*s>%*s|%4" UVuf "| ",
```

Comment:
Seems reasonable to me, but doesn't that need a corresponding cast to UV on the argument as well?

Reply:
Ugh, yes it does, apologies for missing that.